static/templates/user_group_settings/user_group_creation_form.hbs

```diff
@@ -0,0 +1,35 @@
+<div class="hide" id="user-group-creation" tabindex="-1" role="dialog"
+  aria-label="{{t 'User group creation' }}">
+    <form id="user_group_creation_form">
+        <div class="alert user_group_create_info"></div>
+        <div id="user_group_creating_indicator"></div>
+        <div class="user-group-creation-body">
+            <section class="block">
+                <label for="create_user_group_name">
+                    {{t "User group name" }}
+                </label>
+                <input type="text" name="user_group_name" id="create_user_group_name"
+                  placeholder="{{t 'User group name' }}" value="" autocomplete="off" />
+                <div id="user_group_name_error" class="user_group_creation_error"></div>
+            </section>
+            <section class="block">
+                <label for="create_user_group_description">
+                    {{t "User group description" }}
```

Comment:
I feel like "User group description" feels overly wordy. @alya thoughts on changing this label and the similar "Stream description" label in "Manage streams" to just "Description"? 

I think there's a case for having the form still use "User group name" / "Stream name" at the very top, since it helps remind you what you're doing -- just having "Name" there could confuse someone new to the product about what they're naming.

Reply:
Sure, I'm happy to have it changed to "Description" as long as we do this for both groups and streams. Looks like it won't require any documentation updates.